lib/policy/userPolicySchema.json

```diff
@@ -0,0 +1,401 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "title": "AWS Policy schema.",
+    "description": "This schema describes user policy per AWS policy grammar rules",
+    "definitions": {
+        "actionItem": {
+            "type": "string",
+            "pattern": "^[^*:]+:([^:])+|^\\*{1}$"
+        },
+        "resourceItem": {
+            "type": "string",
+            "pattern": "^\\*|arn:aws(:(\\*{1}|[a-z0-9\\*\\-]{2,})*){2}:.*$"
```

Comment:
You are right, my wildcard check was masking it. I have updated the regex to check specifically for 5 occurrences of `:` and added test asserting the behavior.